src/nvector/trilinos/nvector_trilinos.cpp

```diff
@@ -224,14 +224,13 @@ void N_VSpace_Trilinos(N_Vector x, sunindextype *lrw, sunindextype *liw)
 /*
  * MPI communicator accessor
  */
-void *N_VGetCommunicator_Trilinos(N_Vector x)
+MPI_Comm N_VGetCommunicator_Trilinos(N_Vector x)
```

Comment:
Trilinos can be built without MPI, so should this be `SUNComm` instead, or do we require that we only link against MPI-enabled Trilinos installations?

Reply:
I believe our interfaces will work with Trilinos configured with MPI enabled/disabled so this one should be an `SUNComm`.